diff --git a/srcpkgs/xfsprogs/template b/srcpkgs/xfsprogs/template
index 3dac38b886..15ac81f365 100644
--- a/srcpkgs/xfsprogs/template
+++ b/srcpkgs/xfsprogs/template
@@ -10,7 +10,7 @@ short_desc="Utilities for managing the XFS filesystem"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="LGPL-2.1"
 homepage="http://xfs.org/"
-distfiles="http://ftp.kernel.org/pub/linux/utils/fs/xfs/${pkgname}/${pkgname}-${version}.tar.xz"
+distfiles="https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/${pkgname}-${version}.tar.xz"
 checksum=f1e60a9a54583dba82fa506dd9b59bdec110a968f80f507bf5f93b263af7a4df
 
 do_install() {
